File: help/pt_pt/exercicios_da_ficha_3/Ficha03_003/Ficha03_003.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_003 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Valor de X\t: " );
        double x = Double.valueOf(keyb.nextLine());
        System.out.print("Exponencial \t= " + Math.exp( x ) );
        System.out.print("\nPotencia \t= " + Math.pow( Math.E , x ) );
        System.out.print("\nOperador \t= " + Math.pow( Math.E , x ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 05 20:26

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_003/Ficha03_003.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Valor de X    : " ;
    $x = floatval(isset($_POST["x"])? $_POST["x"] : isset($_GET["x"]) ? $_GET["x"] : 0);
    
    echo $x . "<br/>";// print input 
    echo  "Exponencial     = " . ( exp( $x )  ) ;
    echo  "\nPotencia     = " . ( pow( M_E , $x )  ) ;
    echo  "\nOperador     = " . ( M_E ^ $x  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 05 20:26

?>